Makefile

```
@@ -97,6 +97,7 @@ endif
 
 ALWAYS_OFFLOAD_NODE_STATUS := false
 POD_STATUS_CAPTURE_FINALIZER ?= true
+TARGETPLATFORM := linux/amd64
```

As Alan wrote twice now:
> so long as it only built for the "same as your machine" in the default case

This is not "same as your machine"

Also please properly separate words in capital case: 
```suggestion
TARGET_PLATFORM := linux/amd64
```